src/modules/mavlink/mavlink_receiver.h

```diff
@@ -244,6 +246,7 @@ class MavlinkReceiver : public ModuleParams
 	uORB::Publication<vehicle_odometry_s>			_visual_odometry_pub{ORB_ID(vehicle_visual_odometry)};
 	uORB::Publication<vehicle_rates_setpoint_s>		_rates_sp_pub{ORB_ID(vehicle_rates_setpoint)};
 	uORB::Publication<vehicle_trajectory_waypoint_s>	_trajectory_waypoint_pub{ORB_ID(vehicle_trajectory_waypoint)};
+	uORB::Publication<log_message_s>			_log_message_pub{ORB_ID(log_message)};
```

Comment:
This is super nit-picky, but we've generally been trying to keep these lists sorted alphabetically.